public/friendlycode/js/fc/load-project-files.js

```diff
@@ -0,0 +1,215 @@
+define(["jquery"], function($) {
+  "use strict";
+
+  var Path = Bramble.Filer.Path;
+  var FilerBuffer = Bramble.Filer.Buffer;
+
+  // Taken from http://stackoverflow.com/questions/6965107/converting-between-strings-and-arraybuffers
+  function convertToArrayBuffer(string) {
```

Comment:
For follow-up, this is a great candidate function for moving to a worker, and making async.